template/README.md.jinja

```diff
@@ -1,12 +1,31 @@
-| Package | [![Latest PyPI Version](https://img.shields.io/pypi/v/{{project_slug}}.svg)](https://pypi.org/project/{{project_slug}}/) [![Supported Python Versions](https://img.shields.io/pypi/pyversions/{{project_slug}}.svg)](https://pypi.org/project/{{project_slug}}/) [![Documentation](https://readthedocs.org/projects/{{project_slug}}/badge/?version=latest)](https://{{project_slug}}.readthedocs.io/en/latest/?badge=latest)                                                                                                                                                                              |
-| Meta | [![{{ license }}](https://img.shields.io/pypi/l/{{project_slug}}.svg)](LICENSE) [![Code of Conduct](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](.github/CODE_OF_CONDUCT.md) [![Checked with mypy](https://www.mypy-lang.org/static/mypy_badge.svg)](https://mypy-lang.org/) [![Code Style Black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black) [![Linting: Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/charliermarsh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff) |
-| Automation |                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+|        |        |
+|--------|--------|
+| Package | [![Latest PyPI Version](https://img.shields.io/pypi/v/{{project_slug}}.svg)](https://pypi.org/project/{{project_slug}}/) [![Supported Python Versions](https://img.shields.io/pypi/pyversions/{{project_slug}}.svg)](https://pypi.org/project/{{project_slug}}/) {% if use_rtd %}[![Documentation](https://readthedocs.org/projects/{{project_slug}}/badge/?version=latest)](https://{{project_slug}}.readthedocs.io/en/latest/?badge=latest){% endif %} |
+| Meta   | [![Code of Conduct](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md) |
 
-_{{ package_description }}_
+*TODO: the above badges that indicate python version and package version will only work if your package is on PyPI.
+If you don't plan to publish to PyPI, you can remove them.*
+
+{{ project_name }} is a project that (describe what it does here).
```

Comment:
maybe just `{{ project_description }}` ?

Reply:
sure - i'll revert this to keep it simple!